<launch>
	<node name="right_wheel" pkg="differential_drive" type="wheel_encoder.py" output="screen">
		<param name="pinNameA"           value="P9_11"  />
		<param name="pinNameB"           value="P9_13"  />
		<param name="ticks_per_rotation" value="1224"   />
		<param name="wheel_radius"       value="0.0762" />
	</node>
	<node name="right_motor" pkg="differential_drive" type="motor_pid.py" output="screen">
		<param name="pinNameFwd"           value="P8_8"  />
		<param name="pinNameBwd"           value="P8_10" />
		<param name="pinNameCmd"           value="P9_14" />
		<param name="timeout"              value="5"     />
	</node>
</launch>
